treeherder/etl/taskcluster_pulse/handler.py

```diff
@@ -111,6 +111,23 @@ def validateTask(task):
     taskId = message["payload"]["status"]["taskId"]
     asyncQueue = taskcluster.aio.Queue({"rootUrl": message["root_url"]}, session=session)
     task = (await asyncQueue.task(taskId)) if not taskDefinition else taskDefinition
+
+    # Bug 1590512 - A more general solution is needed to avoid using env variables that
+    # are only available for mobile related tasks (this does not work for fenix)
+    try:
```

Comment:
This block ignotes `android-components` jobs that are not associated to `master` or a `pull request`.

This is to prevent hitting the raise `MissingPushException` error and retrying 10 times.